mobile/ethclient.go

```diff
@@ -23,6 +23,15 @@ import (
 
 	"github.com/ethereum/go-ethereum/core/types"
 	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/ethereum/go-ethereum/rpc"
+)
+
+const (
+	PendingBlockNumber = int(rpc.PendingBlockNumber)
+
+	LatestBlockNumber = int(rpc.LatestBlockNumber)
+
+	EarliestBlockNumber = int(rpc.EarliestBlockNumber)
```

Comment:
These are just declared and not used anywhere in the above code. Why?

Reply:
these are consts for mobile API, so no use in Go code.
I tried to do the same like other consts defined in mobile API, at this file:
https://github.com/kinfoundation/go-ethereum/blob/master/mobile/accounts.go